Game.java

```diff
@@ -58,12 +54,12 @@ public void playGame() {
             board = playerMakesMove();
         }
         gameType.gameIsOver(board, playerDeactive);
+        startNewGame();
     }
 
     public void pickGameMode() {
         gameType.displayAllGameModes();
-        Scanner s = new Scanner(System.in);
-        String input = s.nextLine();
+        String input = gameType.userPickGameMode();
```

Comment:
The validation of the user input below, should also be in ConsoleGame.  Think: would it be necessary if you had a GUI or WebGUI?